### DISCUSSION TITLE
Getting Started on Windows

### DISCUSSION THREAD

**Discussion by mountcomp** [mikaelnousiainen/RS41ng#97] [Q&A]
I have just downloaded and installed Docker Desktop for Windows, and I'm having all sorts of trouble trying to get things running.  Does anyone have a step-by-step guide or video that I might be able to follow-along with?  I was able to create the RS41ng.elf file, but when I attempt to run the next commands, I get messages that openocd is not recognized from the Windows command prompt.

`openocd -f ./openocd_rs41.cfg -c "init; halt; flash protect 0 0 63 off; exit"`

What am I missing here?

**Comment by mountcomp** [comment]
Ok, I figured out that I could use the STM32 ST-LINK Utility to load the .hex file onto the sonde.  That worked.  However, what's the process for changing settings?  Do I have to start from scratch and rebuild a firmware every time?  Also, how do I power this thing off now?  With the native firmware, I could press the button for 3 seconds and it would shut down.  Now it seems I have to pull the battery which will make putting it back inside the styrofoam case much more challenging.

**Reply by iz2fly** [reply]
Well, you have successfully loaded the firmware.
To change, you have to start over.
To turn it off, you have to remove the voltage.
After all, this software does different things than the original. For example, with the original firmware you cannot transmit on HF or 144 MHz using the Si5351.
It is always under development, you have to take what you like and, if necessary, make or suggest changes.
Have fun.

**Reply by mountcomp** [reply]
Need some help again.  I somehow fumbled through this yesterday, but cannot seem to do it again today.  I was able to do the build just fine.  Now, when I'm in C:\RS41ng-main and run "docker run --rm -it -v %cd%:/usr/local/src/RS41ng rs41ng_compiler", I get "docker: invalid reference format: repository name (usr/local/src/RS41ng) must be lowercase" errors.

I seems like I ran it from a different directory yesterday, or changed the path somehow, but whatever combination I try today doesn't seem to be working.  Should I still be in the C:\RS41ng-main directory or a different folder?
